user/migrations/0002_alter_user_managers_remove_user_username_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.1 on 2024-11-13 12:40
+
+from django.db import migrations, models
+import user.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('user', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterModelManagers(
+            name='user',
+            managers=[
+                ('objects', user.models.UserManager()),
+            ],
+        ),
+        migrations.RemoveField(
+            model_name='user',
+            name='username',
+        ),
+        migrations.AlterField(
+            model_name='user',
+            name='email',
+            field=models.EmailField(max_length=254, unique=True, verbose_name='email adress'),
```

Comment:
There's a typo in the `verbose_name` for the email field. It should be 'email address' instead of 'email adress'. Please correct this to ensure clarity and consistency.